package.json

```diff
@@ -26,6 +26,7 @@
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
     "@prisma/client": "^6.0.1",
+    "class-transformer": "^0.5.1",
```

Comment:
Why do you need this dependency?

Reply:
because is missing when you run:
- npm install 
-  npm run start:dev 

![image](https://github.com/user-attachments/assets/2295d7a3-41f3-4416-a9b1-9d1fe0e7106f)